modular_skyrat/modules/airlock_override/code/airlock_override.dm

```diff
@@ -163,3 +163,15 @@ GLOBAL_VAR_INIT(force_eng_override, FALSE)
 	engineering_override = TRUE
 	normalspeed = FALSE
 	update_appearance()
+
+/// Make the airlock unrestricted as a temporary emergency exit. arg: duration until it reverts to normal
```

Comment:
For autodoc comments, for adding information about the arguments, I'd like if we tried to keep up with the standard, which is using a comment block, and then having something like this:
```suggestion
/**
 * Make the airlock unrestricted as a temporary emergency exit.
 *
 * Arguments:
 * * duration - How long the airlock will be on temporary emergency access for.
 */```
(just double-check in the code, look for `* Arguments:` and you should find plenty of examples in files written by me, there might be a difference or two because I don't have the repo opened right now)